cypress/e2e/signIn.cy.js

```diff
@@ -2,10 +2,37 @@
 
 describe('Sign In page', () => {
   beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it('Login with valid creds', () => {
+    cy.loginUser('tomsmith', 'SuperSecretPassword!');
+
+    cy.get('.flash.success')
+      .should('contain.text', 'You logged into a secure area!');
+  });
+
+  it('Login with invalid password', () => {
+    cy.loginUser('tomsmith', 'invalidPassword');
```

Comment:
```suggestion
    cy.loginUser('tomsmith', 'invalidPassword');
```
create separate constants username and password. You use them at least twice in the test suite